### ISSUE TITLE
Accept custom colormap

### ISSUE THREAD

**Issue by rb643** [okomarov/schemaball#1]
I would like to use full **custom** color map instead of two extremes. 

I'm currently using [cbrewer](http://uk.mathworks.com/matlabcentral/fileexchange/34087-cbrewer---colorbrewer-schemes-for-matlab) to generate colour schemes and would like to use the 'RdBu' diverging pallet to plot positive and negative values.

![capture](https://cloud.githubusercontent.com/assets/3731173/21964501/142a4a82-db45-11e6-8b9c-a25d3c2d2da6.PNG)

In other words, I would like:

```
map = cbrewer('div', 'RdBu', 20);
schemaball(a,[],map)
```
